File: math/excel-column-title.cpp
string Solution::convertToTitle(int A) {
    string ans="";
    while(A){
        int j = A%26;
        A = A/26;
        if(j==0){ ans += 'Z'; A--;}
        else ans += 'A' + j-1;
    }
    reverse(ans.begin(), ans.end());
    return ans;
}

File: math/excel-column-title.py
class Solution:
    def convert(self, x):
        if x == 0: return 'Z'
        return chr(x + ord('A') - 1)

    def convertToTitle(self, A):
        ans = ""
        while A:
            r = A % 26
            A //= 26
            ans = self.convert(r) + ans
            if r == 0: A -= 1

        return ans
